lte/cloud/go/services/subscriberdb/digest.go

```diff
@@ -22,6 +22,7 @@ import (
 	"magma/orc8r/cloud/go/mproto"
 	mproto_protos "magma/orc8r/cloud/go/mproto/protos"
 	"magma/orc8r/cloud/go/services/configurator"
+	orc8r_protos "magma/orc8r/lib/go/protos"
```

Comment:
No need to alias this one (?)

Reply:
I think this is aliased because the `protos` in this file refers to the `subscriberdb` service-specific protos @hcgatewood